cypress/fixtures/network/requests.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
If this file is not in use, then remove.

Reply:
Removed